modules/security/src/main/DisposableEmailDomain.scala

```diff
@@ -148,6 +148,7 @@ private object DisposableEmailDomain:
       "protonmail.ch",
       "pm.me",
       "proton.me",
+      "passfwd.com", // proton.me
```

Comment:
it's an email provided for Proton Pass users
but the SimpleLogin MX should have allowed it since aliases stay in SimpleLogin